app/src/androidTest/java/com/duckduckgo/app/browser/BrowserChromeClientTest.kt

```diff
@@ -78,7 +78,7 @@ class BrowserChromeClientTest {
         mockFileChooserParams = mock()
         testee.webViewClientListener = mockWebViewClientListener
         webView = TestWebView(getInstrumentation().targetContext)
-        whenever(mockDrm.getDrmPermissionsForRequest(any(), any())).thenReturn(arrayOf())
+        whenever(mockDrm.isDrmAllowedForUrl(any())).thenReturn(false)
```

Comment:
In some of these tests we are validating that grant() was never called, i.e. `verify(mockPermission, never()).grant(any())`. Now that we are denying them we should also validate that deny() was called too.